src/analyticsTracking/AnalyticsViewConsumer.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Amplitude, LogOnMount} from '@amplitude/react-amplitude';
+import useAnalyticsTracking from './useAnalyticsTracking';
+
+/**
+ * View specific AmplitudeProvider consumer component to
+ * log events on each view load and pass inherited props down to views
+ */
+const AnalyticsViewConsumer = ({
```

Comment:
agreed, moved the tracking to the Routes level. Also removed all `VIEW_MOUNT` events to reduce noise in downstream events analysis for now. However, it can be useful to see when people hit and page and _bounce_ without interacting as it can signal problems with IA of the content.